cmake/garant_nightly.cmake

```diff
@@ -1,38 +0,0 @@
-set(CTEST_SOURCE_DIRECTORY "$ENV{HOME}/Dashboards/girder")
```

Comment:
Sadly, I haven't gotten around to setting it up again.  If I need the files, I can get them from the history.